package.json

```diff
@@ -95,5 +95,9 @@
     "vite": "4.3.1",
     "vue": "^3.2.45",
     "vue-tsc": "^1.2.0"
+  },
+  "dependencies": {
+    "@mdi/js": "^7.2.96",
```

Comment:
These should be devDependencies since we don't want them added into the `node_modules` of anything downstream like app or rc